lib/Themes/themes.dart

```diff
@@ -22,29 +23,13 @@ extension ThemeExtension on ThemeData {
 // DIMENSIONS:
 
   double get kCardBorderRadius => 10.0;
-  double get kTextFieldBorderRadius => 8.0;
-  double get kButtonBorderRadius => 8.0;
-
-// FONTS AND TEXT STYLES:
-
-  TextStyle get kTitleTextStyle => GoogleFonts.roboto(
-      fontWeight: FontWeight.w600, color: colorLevel4, fontSize: 20, height: 1);
 

```

Comment:
don't import raleway font explicitly, GoogleFonts.raleway to use raleway font, add the lines that you have deleted fot TextStyle